Resources/Prototypes/Nyanotrasen/Entities/Objects/Consumable/Food/Baked/pizza.yml

```
@@ -24,6 +24,7 @@
   - type: Tag
     tags:
     - Pizza
+    - Vegetable # Yes, I am aware that corn is technically a fruit, but the tagging system is enough of a mess as it is
```

yamllint

[warning] too few spaces before comment

(comments)